docs/tutorials/music.rst

```diff
@@ -47,34 +44,24 @@ MicroPython has quite a lot of built-in melodies. Here's a complete list:
 * ``music.POWER_UP``
 * ``music.POWER_DOWN``
 
-Take the example code and change the melody. Which one is your favourite? How
-would you use such tunes as signals or cues?
+Pobierz któryś z przykładowych kodów i odtwórz melodię. Która podoba Ci się najbardziej? Masz pomysł na ich wykorzystanie jako sygnały?
 
-Wolfgang Amadeus Microbit
-+++++++++++++++++++++++++
+Wolfgang Amadeusz Microbit
+++++++++++++++++++++++++++
 
-Creating your own tunes is easy!
+Zaprogramowanie swojej własnej melodii jest bardzo łatwe!
 
-Each note has a name (like ``C#`` or ``F``), an octave (telling MicroPython how
-high or low the note should be played) and a duration (how
-long it lasts through time). Octaves are indicated by a number ~ 0 is the
-lowest octave, 4 contains middle C and 8 is about as high as you'll ever need
-unless you're making music for dogs. Durations are also expressed as numbers.
-The higher the value of the duration the longer it will last. Such
-values are related to each other - for instance, a duration of ``4`` will last
-twice as long as a duration ``2`` (and so on). If you use the note name ``R``
-then MicroPython will play a rest (i.e. silence) for the specified duration.
+Każda nuta ma nazwę (np. ``C#`` lub ``F``), oktawę (określającą jak wysoko lub nisko dźwięk ma być grany) oraz długość (przez jaki czas dźwięk ma być grany). Oktawy są oznaczone liczbami ~ 0 oznacza najniższą oktawę, oktawa 4 zawiera środkowe C, a 8 jest tak wysoko, że wyższej już nie będziesz potrzebować, chyba że zechcesz tworzyć muzykę dla psów. Długość dźwięku jest również wyrażana jako liczby. Im większa liczba tym dłużej nuta będzie odtwarzana. Te wielkości są od siebie zależne, np. po wprowadzeniu wartości ``4``, nuta będzie odtwarzana 2 razy dłużej niż po wprowadzeniu ``2`` itd. W miejsce nazwy nuty możesz wpisać ``R``. Wtedy MicroPython nie będzie odtwarzał nut przez ustalony czas.
```

Comment:
Jeśli zamiast nuty wpiszesz "R", to MicroPython w tym miejscu zrobi pauzę (ang. rest) o zadanej długości.